src/popup/utils/notifications.ts

```diff
@@ -0,0 +1,64 @@
+import { fetchJson, postJson } from './helper';
```

Comment:
After spending some time with this PR I feel that the contents of this file should be moved to the composable. I know that wouldn't be the perfect idea, but I think still better than storing these functions here. Most of these functions requires a lot of arguments, which are present within the notifications composable. Like for example the SDK.